lib/tarfiles.py

```diff
@@ -73,8 +73,8 @@ def do_tarballs(args):
     we convert the argument to the next type as we go...
     """
 
-    NUM_RETRIES_UPLOAD = 20
-    RETRY_INTERVAL_SEC = 30
+    NUM_RETRIES_UPLOAD = os.getenv("JOBSUB_NUM_RETRIES_UPLOAD", 20)
+    RETRY_INTERVAL_SEC = os.getenv("JOBSUB_RETRY_INTERVAL_SEC", 30)
```

Comment:
These need to be converted to ints, and bad input handled (that is the downside of making things configurable...).

Also, `JOBSUB_RETRY_INTERVAL_SEC` is a bit too generic for my taste, I'd suggest starting both with `JOBSUB_UPLOAD_`.

Reply:
Done.